integration-tests/actions/automation_ocr_helpers.go

```diff
@@ -123,13 +123,23 @@ func CreateOCRKeeperJobs(
 	registryAddr string,
 	chainID int64,
 	keyIndex int,
+	registryVersion ethereum.KeeperRegistryVersion,
 ) {
 	l := utils.GetTestLogger(t)
 	bootstrapNode := chainlinkNodes[0]
 	bootstrapP2PIds, err := bootstrapNode.MustReadP2PKeys()
 	require.NoError(t, err, "Shouldn't fail reading P2P keys from bootstrap node")
 	bootstrapP2PId := bootstrapP2PIds.Data[0].Attributes.PeerID
 
+	var contractVersion string
+	if registryVersion == ethereum.RegistryVersion_2_1 {
+		contractVersion = "v2.1"
+	} else if registryVersion == ethereum.RegistryVersion_2_0 {
```

Comment:
fwiw, I'm not sure we even need to keep integration tests for 2.0 around... since we will not be using them in prod anywhere. I would vote to keep the tests simpler (and faster) and just gut 2.0. wdyt @anirudhwarrier?